types/ckeditor4/index.d.ts

```diff
@@ -1669,6 +1669,7 @@ declare namespace CKEDITOR {
         hidpi?: boolean | undefined;
         lang?: string | string[] | undefined;
         requires?: string | string[] | undefined;
+        icons?: string | undefined;
 
         afterInit?(editor: editor): any;
```

Comment:
Could maybe also add [isSupportedEnvironment](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_pluginDefinition.html#method-isSupportedEnvironment)?

Reply:
Sure this would require a new version right? As it was added in 4.12

Reply:
Updated the PR with isSupportedEnvironment